### PR TITLE
Add home and about pages to repository

### PR DESCRIPTION
**Pull request by c00p75**
- Set up the repository and tools.
- Create the 2 pages for mobile and desktop versions.
- Implement the user interactions: link, mobile menu.
- Implement the section "Featured speakers" with dynamic HTML.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="socials container-fluid bg-dark text-light d-md-flex d-none justify-content-end px-3 gap-4 px-5">
+        <a href="#"><img src="Images/facebook.png" alt="Facebook"></a>
+        <a href="#"><img src="Images/twitter.png" alt="twitter"></a>
+        <span>English</span>
+      </div>
```
Comment (ridwanediallo) [inline home.html]: - [ ] Hey @c00p75, you did an awesome job on this project 👏. However, I have noticed that the items in the `top nav bar` is not aligned to the `right ` with those in the bottom nav bar. And compared to the design there are some items that are remaining. So, what do you think about fixing that? 🤔

### Your design 👇️
Screenshot from 2022-11-26 02-04-40

### The original design 👇️
Screenshot from 2022-11-26 02-04-00

File: home.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="IBM"></div>
+          <a class="prev text-decoration-none d-none d-md-block">❮</a>
+          <a class="next text-decoration-none d-none d-md-block">❯</a>
+        </div>
+      </section>
+    </div>
+    <footer class="lato container-fluid text-dark d-flex justify-content-center pt-3"> 
+        <img src="Images/logo.png" alt="Penda logo" class="px-2 pt-1 d-md-none">
+        <p class="text-dark px-2 pt-md-3" href="#">Penda Inernational Leadership Conference. Some rights reserved &copy2023</p>
+    </footer>
```
Comment (ridwanediallo) [inline home.html]: - [x] I think your `footer` should have your logo aligned to the left and text aligned to the right, but for the desktop view, we can't see the logo. Could you please fix that? 🤔

### your footer 
Screenshot from 2022-11-26 02-21-33

File: about.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_ 13.png" alt="Samsung"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_6.png" alt="Fenty beauty"></div>     
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_9.png" alt="Microsoft"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_10.png" alt="IBM"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_ 14.png" alt="IBM"></div>
+        <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="IBM"></div>
+        <a class="prev text-decoration-none d-none d-md-block">❮</a>
+        <a class="next text-decoration-none d-none d-md-block">❯</a>
+      </div>
+    </section>
```
Comment (ridwanediallo) [inline about.html]: - [x] I think for the desktop view, your need to display the `partners section`. One thing you can do is to hide it for the desktop view.

File: home.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+          </div>          
+        </div>
+      </nav>              
+    </header>
+    
+    <div class="pt-md-5">
+      <main class="container-1 section-container text-center text-md-start">
+        <div class="container-2 container d-flex row gap-1 p-5">
+          <p class="lato quote text-center container-fluid fs-5">"A genuine leader is not a searcher for consensus but a molder of consensus" ~ Martin Luther King, Jr.</p>
+          <h1 class="py-2 h1-effect"><span>PENDA LEADERSHIP CONFERENCE <br>2023</span></h1>
```
Comment (ridwanediallo) [inline home.html:67]: - [x] Please I would recommend you increase the `font size` of your `h1` by giving it something like `5em`. 👌

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @c00p75 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
```
Comment (pinkmoon25) [inline about.html]: - [ ] As requested by the previou reviewer, https://github.com/c00p75/Capstone-project-Conference-page/pull/1#discussion_r1032726920 Please hide the partners section in mobile view for the homepage it should be present in the about page in mobile view see the design 
image

And it is opposite for the desktop, you have to display partners setion in the home page and not in the about page , Please see the[ design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) . Currently you have the partner section visible in both home and about page in both desktop and mobile view, I hope I was able clear thing up :pray:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @c00p75 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
you have done really well with the capstone project :clap: , your coding style is also impressive kudos!, also you have implemented the changes gracefully :clap: I think you missed one, but I have mentioned in the comments below :+1: 



 ## Highlights
Correct github flow :heavy_check_mark: 
Linters checkds :heavy_check_mark: 
You documented your work professionally. :heavy_check_mark:
followed best practices :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1,2 +1,105 @@
...
+
+- Project from [Microverse](https://www.microverse.org/?grsf=i6yi2m) html, css & Javascript module
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (1GORDON) [inline README.md:105]: Great job @c00p75 🤝

- [ ] [Optional] Kindly fix your MIT license link to make your readme file more professional.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @c00p75 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**